tests/cases/fourslash/semanticModernClassificationInfinityAndNaN.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path="fourslash.ts"/>
+
+//// Infinity;
+//// NaN;
+
+verify.semanticClassificationsAre("2020");

```

Comment:
Shouldn't this get a baseline generated?

Reply:
I thought this (and validated it) but the point is that it is ignored, so the lack of any classifications is actually a correct check